### PR TITLE
More SEO text rendered

### PR DESCRIPTION
**Pull request by raduvultur**
Added meta_descriptions for articles and tag pages.
Also changed article title from H1 to H2 since too many H1 on the same page lowers SEO score.

### CODE REVIEW

File: index.hbs [hunk 1]
```diff
@@ -59,9 +59,12 @@ into the {body} of the default.hbs template --}}
                           <span>{{t "Featured"}}</span>
                         </div>
                         <a href="{{url}}" class="m-featured-article__content">
-                          <h1 class="m-featured-article__title js-featured-article-title" title="{{title}}">
+                          <h2 class="m-featured-article__title js-featured-article-title" title="{{title}}">
```
Comment (raduvultur) [inline index.hbs:62]: H1 replaced by H2

File: index.hbs [hunk 2]
```diff
@@ -59,9 +59,12 @@ into the {body} of the default.hbs template --}}
...
                             {{title}}
-                          </h1>
+                          </h2>
+                          {{#if meta_description}}
```
Comment (eddiesigner) [inline index.hbs:65]: The goal of this theme is to be as clean as possible, that's why I don't show the description or the excerpt of posts, if you want to show it you can do it on your own instance of course but I'm afraid that it's something that I won't implement so please remove this part from this PR.

File: partials/loop.hbs [hunk 3]
```diff
@@ -36,6 +36,9 @@
           <h2 class="m-article-card__title js-article-card-title {{#unless feature_image}}js-article-card-title-no-image{{/unless}}" title="{{title}}">
             {{title}}
           </h2>
+          {{#if meta_description}}
```
Comment (eddiesigner) [inline partials/loop.hbs:39]: Same comment as above, also note that might be better to use `excerpt` instead of `meta_description`

File: tag.hbs [hunk 4]
```diff
@@ -16,17 +16,11 @@ into the {body} of the default.hbs template --}}
...
+        <div class="m-tag__description">{{ description }}</div>
       {{/if}}
-      <div class="m-hero__meta">
-        <ul class="m-hero-stats">
-          <li>
-            {{plural ../pagination.total empty=(t "No posts") singular=(t "1 post") plural=(t "% posts")}} {{t "with this tag"}}
-          </li>
-        </ul>
-      </div>
+      <p class="m-heading__description">
```
Comment (eddiesigner) [inline tag.hbs:21]: Please keep it like it is now, there are some scenarios where the class `m-heading__description` doesn't work here.

File: partials/loop.hbs (hunk 3 above)
Comment (raduvultur) [reply partials/loop.hbs:39]: Thanks for the `excerpt` tip

File: index.hbs (hunk 2 above)
Comment (raduvultur) [reply index.hbs:65]: I think that you are perfectly right. I also like it clean. SEO made me do it :)
Please consider the switch from H1 to H2.

**Pull request closed without merging**